tests/TypeApiTests.hs

```diff
@@ -43,26 +43,26 @@ getQuery query =
         parseOnly (document <* endOfInput) query
   in selectionSet
 
-runQuery :: AST.SelectionSet -> IO (Either Text Value)
+runQuery :: AST.SelectionSet -> IO (Either Text Result)
```

Comment:
It's an error because we run over a custom `ExceptT` monad (only in this specific test though)